### PR TITLE
change the appearance of report screen 

### PR DESCRIPTION
**Pull request by Allof02**
only changed a few codes associated to ui appearence

### CODE REVIEW

**Review by foreverwoods — APPROVED** [review]
looks fine, just GUI changes

File: README.md [hunk 1]
```diff
@@ -1,39 +1,62 @@
-# Project Template
+# Project Group 72
+* do we have a name for the app?
```
Comment (prajogt) [inline README.md:2]: I think we decided on 72Reviews

**Review by prajogt — APPROVED** [review]
Looks good

File: README.md (hunk 1 above)
Comment (Allof02) [reply README.md:2]: yeah, we can just edit it on the main branch after merging this PR

**Review by JarodGratton — APPROVED** [review]

**Pull request merged**